From ef6fe92876cf6b9a572457896bb8c420a0cceba4 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Mon, 27 Mar 2017 13:11:34 +0200
Subject: [PATCH] man: update doc after support of ESN and anti-replay window

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1425059
Upstream Status: iproute2.git commit eeb669a7403cb

commit eeb669a7403cbca859070aaed6f1a55131d75627
Author: Nicolas Dichtel <nicolas.dichtel@6wind.com>
Date:   Thu Oct 30 01:18:48 2014 -0700

    man: update doc after support of ESN and anti-replay window

    Signed-off-by: Nicolas Dichtel <nicolas.dichtel@6wind.com>
---
 man/man8/ip-xfrm.8 | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/man/man8/ip-xfrm.8 b/man/man8/ip-xfrm.8
index aa04804..11f7104 100644
--- a/man/man8/ip-xfrm.8
+++ b/man/man8/ip-xfrm.8
@@ -43,6 +43,10 @@ ip-xfrm \- transform configuration
 .IR SEQ " ]"
 .RB "[ " replay-oseq
 .IR SEQ " ]"
+.RB "[ " replay-seq-hi
+.IR SEQ " ]"
+.RB "[ " replay-oseq-hi
+.IR SEQ " ]"
 .RB "[ " flag
 .IR FLAG-LIST " ]"
 .RB "[ " sel
@@ -140,7 +144,8 @@ ip-xfrm \- transform configuration
 
 .ti -8
 .IR FLAG " :="
-.BR noecn " | " decap-dscp " | " nopmtudisc " | " wildrecv " | " icmp " | " af-unspec " | " align4
+.BR noecn " | " decap-dscp " | " nopmtudisc " | " wildrecv " | " icmp " | "
+.BR af-unspec " | " align4 " | " esn
 
 .ti -8
 .IR SELECTOR " :="
@@ -498,7 +503,7 @@ and inbound trigger
 .I FLAG-LIST
 contains one or more of the following optional flags:
 .BR noecn ", " decap-dscp ", " nopmtudisc ", " wildrecv ", " icmp ", "
-.BR af-unspec ", or " align4 "."
+.BR af-unspec ", " align4 ", or " esn "."
 
 .TP
 .IR SELECTOR
-- 
1.8.3.1

